java/src/jmri/jmrit/signalsystemeditor/SignalMast.java

```diff
@@ -0,0 +1,140 @@
+package jmri.jmrit.signalsystemeditor;
+
+import java.util.*;
+
+/**
+ * A signal mast type
+ * @author Daniel Bergqvist (C) 2022
+ */
+public class SignalMast {
```

Comment:
Maybe this class should be SignalMastType or MastType.  `SignalMast` which refers to the appearance file can be confused with `SignalMast` which refers to a jmri.SignalMast object.

Reply:
Thanks. I have renamed `SignalMast` to `SignalMastType` and `SignalMastXml` to `SignalMastTypeXml` in PR #11651 which replaces this PR.